projects/demo-integrations/src/tests/kit/time/time-step.cy.ts

```diff
@@ -0,0 +1,272 @@
+                        .should('have.value', '23:01');
+                });
+
+                it('{downArrow} => |23 => 23| => type :{upDown} => 23:59', () => {
+                    cy.get('@input')
+                        .type('{downArrow}')
+                        .should('have.value', '23')
+                        .should('have.a.prop', 'selectionStart', 0)
+                        .should('have.a.prop', 'selectionEnd', 0)
+                        .type(`${'{rightArrow}'.repeat(2)}:{downArrow}`)
+                        .should('have.value', '23:59');
+                });
+
+                it('{upArrow}*24 => 00', () => {
+                    cy.get('@input')
+                        .type('{upArrow}'.repeat(24))
+                        .should('have.value', '00');
+                });
+
+                it('type {upArrow}*12 => |12 => 1|2 => type {upArrow} => 1|3 => 13| => type {downArrow} => 12|', () => {
+                    cy.get('@input')
+                        .type('{upArrow}'.repeat(12))
+                        .should('have.value', '12')
+                        .should('have.a.prop', 'selectionStart', 0)
+                        .should('have.a.prop', 'selectionEnd', 0)
+                        .type('{rightArrow}')
+                        .should('have.a.prop', 'selectionStart', 1)
+                        .should('have.a.prop', 'selectionStart', 1)
+                        .type('{upArrow}')
+                        .should('have.value', '13')
+                        .should('have.a.prop', 'selectionStart', 1)
+                        .should('have.a.prop', 'selectionStart', 1)
+                        .type('{rightArrow}')
+                        .should('have.a.prop', 'selectionStart', 2)
+                        .should('have.a.prop', 'selectionStart', 2)
+                        .type('{downArrow}')
+                        .should('have.value', '12')
+                        .should('have.a.prop', 'selectionStart', 2)
+                        .should('have.a.prop', 'selectionStart', 2);
+                });
+
+                it('type 12:{upArrow}{rightArrow}{downArrow}{rightArrow}{downArrow} => 12:59', () => {
+                    cy.get('@input')
+                        .type(
+                            '12:{upArrow}{rightArrow}{downArrow}{rightArrow}{downArrow}',
+                        )
+                        .should('have.value', '12:59');
+                });
+            });
+
+            describe('step = 0', () => {
+                beforeEach(() => {
+                    cy.visit(`/${DemoPath.Time}/API?mode=HH:MM&step=0`);
+                    cy.get('#demo-content input')
+                        .should('be.visible')
+                        .first()
+                        .focus()
+                        .clear()
+                        .as('input');
+                });
+
+                it('should be disabled', BROWSER_SUPPORTS_REAL_EVENTS, () => {
```

Comment:
Could we use built-in cypress `{upArrow}` / `downArrow` ?

Reply:
No we can't. If we'll use built-in, it will not trigger real browser `upArrow` event